src/ur5e_arm.cpp

```diff
@@ -403,6 +408,47 @@ void UR5eArm::send_trajectory(const std::vector<vector6d_t>& p_p,
 
 // helper function to read a data packet and send a noop message
 void UR5eArm::read_and_noop() {
+    std::string status;
+    dashboard->commandSafetyStatus(status);
+    if (status.find(urcl::safetyStatusString(urcl::SafetyStatus::NORMAL)) == std::string::npos) {
```

Comment:
we assume any non normal safety status is an estop. this might technically be incorrect